ideas/313-sticker-market/readme.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: #312-janitors
-title: Swarm Janitors
+id: #313-sticker-market
+title: Sticker Market
 status: research
```

Comment:
Could also change status while we're here...!

Reply:
Done!